sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -321,7 +325,9 @@ def fit(self, X, y, sample_weight=None):
         n_threads = _openmp_effective_n_threads()
 
         if isinstance(self.loss, str):
-            self._loss = self._get_loss(sample_weight=sample_weight)
+            self._loss = self._get_loss(
+                sample_weight=sample_weight, quantile=self.quantile
+            )
```

Comment:
I do not think we do not need to change the interface for `_get_loss`. The subclass `HistGradientBoostingRegressor` passes `self.quantile` directly into `PinballLoss`.

Furthermore, I do not think `BaseHistGradientBoosting` needs a `quantile` parameter. Only `HistGradientBoostingRegressor` requires it.